workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -292,3 +292,19 @@ def db_schema_get_current_revision():
     ])
     revision = output.split(' ', 1)[0]
     return revision
+
+
+def stage_db_schema_get_current_revision():
```

Comment:
makes sense, but i wanted it to be compatible with the other funcs (db_schema_...)